contracts/script/SetupPayments.s.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.12;
+
+import {Script} from "forge-std/Script.sol";
+import {IncredibleSquaringDeploymentLib} from "./utils/IncredibleSquaringDeploymentLib.sol";
+import {CoreDeploymentLib} from "./utils/CoreDeploymentLib.sol";
+import {SetupPaymentsLib} from "./utils/SetupPaymentsLib.sol";
+import {IRewardsCoordinator} from "@eigenlayer/contracts/interfaces/IRewardsCoordinator.sol";
+import {console2} from "forge-std/console2.sol";
+import {UpgradeableProxyLib} from "./utils/UpgradeableProxyLib.sol";
+import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+interface Ownable {
+    function owner() external view returns (address);
+}
+
+contract SetupPayments is Script {
+    struct PaymentInfo {
+        address[] earners;
+        bytes32[] earnerTokenRoots;
+        address recipient;
+        uint256 numPayments;
+        uint256 amountPerPayment;
+        uint32 duration;
+        uint32 startTimestamp;
+        uint32 endTimestamp;
+        uint256 indexToProve;
+    }
+
+    address private deployer;
+    CoreDeploymentLib.DeploymentData coreDeployment;
+    IncredibleSquaringDeploymentLib.DeploymentData incredibleSquaringDeployment;
+    string internal constant paymentfilepath = "test/mockData/scratch/payments.json";
+    string internal constant filePath = "test/mockData/scratch/payment_info.json";
+
+    uint256 constant NUM_TOKEN_EARNINGS = 1;
+    uint256 constant DURATION = 1 weeks;
+
+    function setUp() public {
+        deployer = vm.rememberKey(vm.envUint("PRIVATE_KEY"));
+        vm.label(deployer, "Deployer");
+
+        coreDeployment = CoreDeploymentLib.readDeploymentJson("script/deployments/core/", block.chainid);
+        incredibleSquaringDeployment =
+            IncredibleSquaringDeploymentLib.readDeploymentJson("script/deployments/incredible-squaring/", block.chainid);
+    }
+
+    function run() external {
+        vm.startBroadcast(deployer);
+        uint256 amount_per_payment = vm.parseJsonUint(vm.readFile(filePath), ".amountPerPayment");
```

Comment:
Add an import
`import {stdJson} from "forge-std/StdJson.sol";`

add using syntax below the contract definition
`using stdJson for string;`

read the file once 
`string memory json = vm.readFile(filePath);`

parse the json
`uint256 amountPerPayment = json.readUint(".amountPerPayment");`